opal-core-ws/src/main/java/org/obiba/opal/web/project/ProjectResource.java

```diff
@@ -139,6 +138,11 @@ public VCFStoreResource getVCFStoreResource() {
     }
     VCFStoreResource resource = applicationContext.getBean(VCFStoreResource.class);
     resource.setVCFStore(project.getVCFStoreService(), name);
+    resource.setVCFSamplesSummaryBuilder(
+      new VCFSamplesSummaryBuilder()
+        .datasource(project.getDatasource())
```

Comment:
Why is the datasource the one of the project? There is no reason for such a restriction. The VCF samples mapping should use a table reference instead.